src/types/platforms-types/nina.ts

```diff
@@ -55,16 +53,17 @@ const mapArtistProfile = ({ apiTrack, nft, contract }: { apiTrack: any, nft?: NF
     platformId: nft.platformId,
     avatarUrl: undefined,
     websiteUrl: `${nft.metadata.external_url}/related`,
-    createdAtTime: nft.createdAtTime, 
+    createdAtTime: nft.createdAtTime,
   }
 };
 
 const mapNFTtoTrackID = (nft: NFT): string => {
+  // TODO: extract solanaTrackId function and provide appropriate trackId
```

Comment:
we don't really use TODOs in our codebase, and probably shouldn't start

also this is minor enough that i wouldn't create a github issue or anything to track it, so just leave as is i think

```suggestion
```